### PR TITLE
corrected the conflict merges, added in the changes that were require…

### PR DESCRIPTION
**Pull request by AlanSimpson1723**
…d, added the option to review your moves on choice at the end of the game then allows to to either play again. Made Public static final scanner, moved the rules into their own method

### CODE REVIEW

**Review by EightFold24 — APPROVED** [review]
looks good

File: .idea/Elevens.iml [hunk 1]
```diff
@@ -12,6 +12,16 @@
...
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-module-junit4:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-module-junit4-common:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-core:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.javassist:javassist:3.21.0-GA" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-reflect:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-api-mockito:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.mockito:mockito-core:1.10.19" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.objenesis:objenesis:2.1" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-api-mockito-common:1.6.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.powermock:powermock-api-support:1.6.6" level="project" />
```
Comment (JordanGibson) [inline .idea/Elevens.iml:24]: None of this should be changed

File: .idea/misc.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="false" project-jdk-name="15" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/../../OpenIV/out" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_12" default="false" project-jdk-name="15" project-jdk-type="JavaSDK">
+    <output url="file://$USER_HOME$/OpenIV/out" />
   </component>
```
Comment (JordanGibson) [inline .idea/misc.xml:12]: Same here

File: src/main/model/Game.java [hunk 3]
```diff
@@ -141,4 +173,48 @@ protected void removeCardFromBoard(Card card) {
...
+
+        }
+        if(isStalemate()) {
+            System.out.println("Unfortunately this time you have lost! :( \n");
+            movesPlayback();
+        }
+    }
+
+    public void movesPlayback(){
+        System.out.println("would you like to see your moves back? (y/n) : ");
```
Comment (JordanGibson) [inline src/main/model/Game.java:190]: `Would you like to see a replay?` is probably better English imo

**Review by JordanGibson — COMMENTED** [review]
Looks good overall, just a few minor comments

File: .idea/Elevens.iml (hunk 1 above)
Comment (AlanSimpson1723) [reply .idea/Elevens.iml:24]: i didnt even know i changed it

File: src/main/model/Game.java (hunk 3 above)
Comment (AlanSimpson1723) [reply src/main/model/Game.java:190]: > `Would you like to see a replay?` is probably better English imo

that can be something we can add in at the end sure. it wont brake anything tbf but you are right

**Review by EightFold24 — APPROVED** [review]
he fixer uppered it

**Review by JordanGibson — APPROVED** [review]
good stuff

**Pull request merged**